README.md

```diff
@@ -146,7 +80,92 @@ repository, we recommend including it as a project dependency in your
 ```
 
 [Composer]: https://getcomposer.org/
-[Packagist]: https://packagist.org/
+[Packagist]: https://packagist.org/packages/vimeo/ablincoln
+
+### Comparison with Python Release
+
+ABLincoln and the original Python release of PlanOut are very similar in both
+functionality and usage. Both packages implement abstract and concrete versions
+of the Experiment and Namespace classes, parameter overrides to facilitate
+testing, exposure logging systems, and various random assignment operators
+tested and confirmed to produce identical outputs.
+
+Notable differences between the two releases currently include:
+  - ABLincoln features native support for logging either to local files or
+  to any PSR-compliant stack through the use of included PHP logging traits
+  - ABLincoln does not currently include an interpreter for the
+  [PlanOut language][].
+
+[PlanOut language]: http://facebook.github.io/planout/docs/planout-language.html
+
+### Usage in Production Environments
+
+ABLincoln was ported and designed with scalability in mind. Here are a few ways
+that Vimeo has chosen to extend it to meet the needs of our testing process:
+
+##### Serialized Experiments
+
```

Comment:
It doesn't appear that any of this functionality is included in the current release so I don't think it makes sense to include it in the README...